site/pages/about-us/principles-slice/index.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import style from './style.css';
+import Principle from './principle';
+import Button from '../../../components/button';
+import Link from '../../../components/link';
+
+export default function Principles() {
+  return (
+    <div className={style.principles}>
+      <div className={style.container}>
+        <div className={style.header}>Our principles.</div>
+        <div className={style.thisIsWhatWeBelieveIn}>
+          This is what we believe – and what the founders say after a couple of pints
+        </div>
+        <Principle number="01" title="A team of leaders." description="We trust our people, giving them the autonomy to make the right decisions." />
```

Comment:
Let's improve the semantics by making it a list, that would help screen readers to interpret this better.